cmd/backup/config.go

```diff
@@ -58,6 +61,14 @@ type Config struct {
 	LockTimeout                time.Duration `split_words:"true" default:"60m"`
 }
 
+func (c *Config) ResolveSecret(envVar string, secretPath string) (string, error) {
+	if secretPath != "" {
+		data, err := os.ReadFile(secretPath)
+		return string(data), err
```

Comment:
Most parts of the code in here will wrap returned errors with some extra info which can be pretty helpful:

```go
return string(data), fmt.Errorf("resolveSecret: error reading secret path: %w", err)
```